src/integration/java/nz/cri/gns/NZSHM22/opensha/hazard/NZSHM22_HazardCalculatorBuilder_IntegrationTest.java

```diff
@@ -37,7 +37,7 @@ public void hazardCalcTest() throws URISyntaxException, DocumentException, IOExc
         calculator = builder.build();
         actual = calculator.calc(-41.289, 174.777);
 
-        expected = Arrays.asList(0.9999999512781396, 0.9999999474424017, 0.99999994162221, 0.9999999302444109, 0.9999999087882951, 0.9999998600470095, 0.9999997637003917, 0.9999995422748154, 0.9999989557156839, 0.9999973559609533, 0.9999923170813609, 0.999976697763896, 0.9999290708652946, 0.9997752153593891, 0.9993395764300738, 0.9981381739135964, 0.9951144421470798, 0.98825099287458, 0.9741731326154526, 0.9484873686258741, 0.9061727267938576, 0.8438768226218849, 0.7610161976896617, 0.6609941629678483, 0.5503482768445943, 0.4381444571271227, 0.3327522518702153, 0.24093446441809752, 0.166335025415553, 0.10960446786846234, 0.0689782492042672, 0.04152875131781453, 0.02394176389656144, 0.01323107292408432, 0.007013933749162593, 0.003567123035584596, 0.0017402483459805218, 8.138434171216513E-4, 3.643516485954956E-4, 1.5582441382777645E-4, 6.34766476617088E-5, 2.4538540094876105E-5, 8.964484364049063E-6, 3.0806694095719678E-6, 9.914190550741608E-7, 2.975497593871168E-7, 8.298536013917612E-8, 2.144889499344771E-8, 5.1292357028387414E-9, 1.1343218586645776E-9, 2.3216539801751424E-10);
+        expected = Arrays.asList(1.0, 1.0, 1.0, 1.0, 1.0, 0.9999999999999999, 0.9999999999999998, 0.9999999999999989, 0.9999999999999925, 0.9999999999999312, 0.9999999999991339, 0.9999999999879503, 0.999999999831085, 0.9999999973983922, 0.9999999665851131, 0.9999996118919805, 0.9999962081168284, 0.9999699607181228, 0.9998081657023239, 0.9990289210478789, 0.9960411527590377, 0.9869816637761328, 0.9648683398418673, 0.9207806104979249, 0.8474196480157911, 0.7442864556904661, 0.6189901266372444, 0.48565138535305696, 0.3592749941799578, 0.25111598411936187, 0.16626604849636029, 0.10467524520049287, 0.06284799790336593, 0.03608342967890055, 0.019846907568932592, 0.010466902673227918, 0.005293722394825617, 0.0025657456413076263, 0.0011898271220269763, 5.266219848827358E-4, 2.2172907727058266E-4, 8.845264707091349E-5, 3.328593366547228E-5, 1.1760905246904763E-5, 3.884275600607445E-6, 1.194272810312036E-6, 3.4065684717532463E-7, 8.991112887546393E-8, 2.192300252445989E-8, 4.936000164157406E-9, 1.0269930461603849E-9);
```

Comment:
As per @voj chat File loading in old form was wrong, some lingering U3 modular artifacts.